Makefile

```diff
@@ -10,7 +22,38 @@ smudge:
 
 smear: smear/libsmear.a
 
-smudge-splat_%.deb:
+stage:
+	rm -rf $(SPLAT_RELEASE_STAGE_DIR)
+	mkdir -p $(SPLAT_RELEASE_STAGE_DIR)
+	cp LICENSE $(SPLAT_RELEASE_STAGE_DIR)
+	cp README.md $(SPLAT_RELEASE_STAGE_DIR)
+
+package: $(foreach EXT,$(PKGEXT),$(PACKAGE)_$(SPLAT_VERSION)_$(PLATFORM).$(EXT))
+
+zip: $(PACKAGE)_$(SPLAT_VERSION)_$(PLATFORM).zip
+$(PACKAGE)_$(SPLAT_VERSION)_$(PLATFORM).zip: stage
+	make -C smudge zip
+	make -C smear zip
+	cd $(SPLAT_RELEASE_STAGE_DIR) && \
+	unzip ../smudge/smudge-*-*.zip && \
+	unzip ../smear/libsmear-dev_*_$(PLATFORM).zip
+	if type zip >/dev/null 2>&1; then \
+	    zip -r $@ $(SPLAT_RELEASE_STAGE_DIR); \
+	elif type 7z >/dev/null 2>&1; then \
+	    7z a $@ $(SPLAT_RELEASE_STAGE_DIR); \
+	fi
+
+tgz: $(PACKAGE)_$(SPLAT_VERSION)_$(PLATFORM).tgz
+$(PACKAGE)_$(SPLAT_VERSION)_$(PLATFORM).tgz: stage
+	make -C smudge tgz
+	make -C smear tgz
+	cd $(SPLAT_RELEASE_STAGE_DIR) && \
+	tar -xf ../smudge/smudge-*-*.tgz && \
+	tar -xf ../smear/libsmear-dev_*_$(PLATFORM).tgz
+	fakeroot tar -czf $@ $(SPLAT_RELEASE_STAGE_DIR)
```

Comment:
So that when root untars it, the default is `tar --same-owner`, which without `fakeroot` leaves the files owned by the UID that `tar`ed the file originally.